tools/CouchDB_Import_MRI.php

```diff
@@ -199,6 +229,110 @@ function _addMRIFeedback($current_feedback, $scan_type, $mri_feedback)
     }
 
     /**
+    * Gets the date according to the type
+    *
+    * @param MRIFile $file  file object
+    * @param string  $type  type of the date
+    * @param array   $array array containing the date
+    *
+    * @return date if exists, if not an empty string
```

Comment:
This should be more explicit about date type format in the documentation. Is it referring to MYSQL YYYY-MM-DD format? QuickForm's array('Y' =>.., 'M' =>.., 'd' =>....) format? Dicom's YYYYMMDD format? PHP's date object? It's hard to tell since it just keeps referring to it as "date"